src/test/ui/rfc-2091-track-caller/track-caller-ffi.rs

```diff
@@ -0,0 +1,50 @@
+// run-pass
+
+#![feature(track_caller)]
+
+use std::panic::Location;
+
+extern "Rust" {
+    #[track_caller]
+    fn tracked() -> &'static Location<'static>;
+    fn untracked() -> &'static Location<'static>;
+}
+
+fn nested_tracked() -> &'static Location<'static> {
+    unsafe { tracked() }
+}
+
+mod provides {
```

Comment:
Hmm maybe you can add a cross-crate or even run-make test, as well.

Reply:
(Resolving per discussions in private.)